app/src/.gitignore

```diff
@@ -0,0 +1,20 @@
+.cache/
```

Comment:
Did you mean to add this in src?

Reply:
Yeah - it seems like git is adding things we don't want, so I wanted to make sure it was getting everything in that folder too.  You can have multiple of them from what I understand.